### DISCUSSION TITLE
Is there a way to prevent "->" and "<-" from being rendered as arrows?

### DISCUSSION THREAD

**Discussion by Mbucari** [AvaloniaUI/Avalonia#9933] [Q&A] [answered]
In Avalonia 11.0.0-Preview-4, These strings render in Text controls as arrows (see below). Is there a style or setting to prevent this and only show characters verbatim?

![image](https://user-images.githubusercontent.com/37587114/211166419-ea103109-5dbc-4a6a-9352-4d808fd1141c.png)

**Comment by kekekeks** [comment, marked as answer]
IIRC that's determined by the font.

**Reply by Mbucari** [reply]
Yup, changing the font from Avalonia's Fluent Inter font to my system's default solved that. Interestingly, I installed the Inter fonts on my machine, and when I type "->" in Microsoft word using that font it does not display as an arrow.

Thanks!

**Reply by rabbitism** [reply]
@Mbucari usually ligature can be turned on and off. But I  don't know how avalonia determines.

**Reply by Gillibald** [reply]
This is currently not possible but planned in the short future
